tests/unit/metadata/test_multi_table.py

```diff
@@ -361,11 +394,7 @@ def test_validate_child_map_circular_relationship(self):
         'sdv.metadata.multi_table.MultiTableMetadata._validate_no_missing_tables_in_relationship'
     )
     @patch('sdv.metadata.multi_table.MultiTableMetadata._validate_relationship_key_length')
-    @patch('sdv.metadata.multi_table.MultiTableMetadata._validate_relationship_sdtypes')
-    @patch('sdv.metadata.multi_table.MultiTableMetadata._validate_missing_relationship_keys')
```

Comment:
why are we changing these but not the other two?

Reply:
I changed those to `instance` methods instead since the previous validation that I was performing wasn't as intended.